openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -216,16 +217,40 @@ def _filter_fields(self, field_whitelist, serialized_account):
         return visible_serialized_account
 
 
+class UserAccountDisableHistorySerializer(serializers.ModelSerializer):
+    """
+    Class that serializes User account disable history
+    """
+
+    disabled_by = serializers.SerializerMethodField()
```

Comment:
if you use `disabled_by` in the models you don't need to provide that field here.

Reply:
disabled_by will only return the user id so I had two options. 1st was to create a Serializer that only returns the username from the model. 2nd was to create a custom field that returns the username instead of the id.